### DISCUSSION TITLE
How should a user of now-archived Tippy.js proceed?

### DISCUSSION THREAD

**Discussion by leafac** [floating-ui/floating-ui#3126] [Q&A]
Hi y’all,

First, thanks for all the work you’ve put into Tippy.js in the past. I’ve used it for many years in many projects and it has always been great to work with 🤗

Today I was visiting the GitHub repository for Tippy.js and noticed that it had been archived.

I didn’t find any explanation, but after a bit of looking around I stumbled upon this repository. As far as I understand, it’s maintained by the people who used to work on Tippy.js.

That’s why I came here to ask:

How do I proceed?

As far as I understand, Floating UI is the new version of Popper, the underlying library upon which Tippy.js was built. But what’s the future of Tippy.js?

Thanks in advance for your answers.

**Comment by FezVrasta** [comment]
Hey 👋 Floating UI includes UI React components that work similarly to Tippy. There's no support for vanilla JavaScript for those UI components though.

**Reply by leafac** [reply]
Thanks for the answer 🤗

Ah, I see. I use vanilla JavaScript, so I didn’t go over the parts of the documentation that cover React 😅

Do you know if something for vanilla JavaScript is planned?

**Reply by FezVrasta** [reply]
Not really. But I wonder if using a lightweight React alternative could be used to run the React bindings on your project. It would probably end up being not much heavier than a first party vanilla library I suppose

**Reply by leafac** [reply]
Sounds good.

Thanks for the information 🤗
